index.ts

```diff
@@ -0,0 +1,355 @@
+/* global Promise */
+import axeCore = require('axe-core');
+import rIC = require('requestidlecallback');
+import after = require('./after');
+
+const requestIdleCallback = rIC.request;
+const cancelIdleCallback = rIC.cancel;
+
+let React;
+let ReactDOM;
+
+const boldCourier = 'font-weight:bold;font-family:Courier;';
+const critical = 'color:red;font-weight:bold;';
+const serious = 'color:red;font-weight:normal;';
+const moderate = 'color:orange;font-weight:bold;';
+const minor = 'color:orange;font-weight:normal;';
+const defaultReset = 'font-color:black;font-weight:normal;';
+
+let idleId: number | undefined;
+let timeout: number;
+let context: axeCore.ElementContext;
+let _createElement: Function;
```

Comment:
`typeof React.createElement` (or something, I don't remember how to do this) might be better here. Using `Function` is pretty ambiguous.